main.py

```diff
@@ -4835,7 +4862,7 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
                     self.message(channel, "Usage: !raritychange <ID> <rarity>", isWhisper)
                     return
 
-                if waifu['base_rarity'] >= int(config['numNormalRarities']):
+                if waifu['base_rarity'] == int(config['numNormalRarities']):
```

Comment:
Do you really want to change this? I know the message says "special", but is there a legitimate use case to un-promo promos?

Reply:
Re-adding them to packs, which functions fine by just having the people who already own the card keep their copy at promo. Just need to disable the stars for promo rarity.

Reply:
That'd be a (possibly massive) hand space penalty for people who rely on promos staying promos, though. Is that really a guarantee you can just drop like this?

Reply:
Re-read my comment - any existing promo copies would stay at promo for the duration that the card was back in packs.